### PR TITLE
API-7198: Fix params in v3.3 customer configuration endpoints

### PR DESCRIPTION
**Pull request by litwinow**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7198-configuration-endpoints-fixes)
- [Jira](https://livechatinc.atlassian.net/browse/API-7198)

## 📓 Description
* whitelist -> allowed
* prechat_survey -> prechat_form
* group_id optional in get_dynamic_configuration

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

## 👷 Type of change (remove not needed)
Update of existing content in dev preview version.

### Content

- Proofreading/content fixes
  
### Features (for the website)

- Breaking change

### CODE REVIEW

File: content/messaging/customer-chat-api/v3.1/index.mdx [hunk 1]
```diff
@@ -2159,8 +2159,7 @@ Returns an empty ticket form of a <a href="https://www.livechatinc.com/help/chat
 | Parameter | Notes                                                                                                                  |
 | --------- | ---------------------------------------------------------------------------------------------------------------------- |
 | `enabled` | If `enabled: false`, the `form` object is not returned. Prechat/postchat survey disabled in the LiveChat Agent App UI. |
-| `header`  | For headers; the field has no `answer`, therefore is not sent in the [Filled form](#filled-form) event.                |
-| `rating`  | For rating; the field has no `answer`, therefore is not sent in the [Filled form](#filled-form) event.                 |
+| `form`    |                                                                                                                        |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.1/index.mdx:2162]:
```suggestion
| `form`    |  The `form` object                                                                                                                    |
```

File: content/messaging/customer-chat-api/v3.1/rtm-reference/index.mdx [hunk 2]
```diff
@@ -2459,8 +2459,7 @@ Returns an empty ticket form of a <a href="https://www.livechatinc.com/help/chat
 | Parameter | Notes                                                                                                                  |
 | --------- | ---------------------------------------------------------------------------------------------------------------------- |
 | `enabled` | If `enabled: false`, the `form` object is not returned. Prechat/postchat survey disabled in the LiveChat Agent App UI. |
-| `header`  | For headers; the field has no `answer`, therefore is not sent in the [Filled form](#filled-form) event.                |
-| `rating`  | For rating; the field has no `answer`, therefore is not sent in the [Filled form](#filled-form) event.                 |
+| `form`    |                                                                                                                        |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.1/rtm-reference/index.mdx:2462]:
```suggestion
| `form`    | The `form` object                                                                                                                     |
```

File: content/messaging/customer-chat-api/v3.1/index.mdx (hunk 1 above)
Comment (ale-kacperczyk) [reply content/messaging/customer-chat-api/v3.1/index.mdx:2162]: I'd add it because it looks empty

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Comment by litwinow** [comment]
@ale-kacperczyk , one more change XD

File: content/messaging/customer-chat-api/v3.3/index.mdx [hunk 3]
```diff
@@ -995,11 +995,12 @@ Returns the dynamic configuration of a given group. It provides data to call [**
...
-| `domain_whitelisted`    | `bool`     | Specifies if `url` is configured as a trusted domain (configurable in the Agent Application).  |
-| `config_version`        | `string`   | The version for which you want to get a configuration. Use it as `version` when calling [**Get Configuration**](#get-configuration). |
-| `localization_version`  | `string`   | The version for which you want to get a localization. Use it as `version` when calling [**Get Localization**](#get-localization). |
+| `group_id`              | `number`   | The ID of the group for which the dynamic configuration is returned.                                                                                                      |
+| `client_limit_exceeded` | `bool`     | Specifies if the limit of customers having a chat was exceeded.                                                                                                           |
+| `domain_whitelisted`    | `bool`     | Specifies if `url` is configured as a trusted domain (configurable in the Agent Application).                                                                             |
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/v3.3/index.mdx:1000]:
```suggestion
| `domain_allowed`    | `bool`     | Specifies if `url` is configured as a trusted domain (configurable in the Agent Application).                                                                             |
```

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**